### ISSUE TITLE
When pdf path contains space,throws error

### ISSUE THREAD

**Issue by xl736592857** [artiebits/pdf-to-printer#67]
This is my code:
```javascript
import printer from 'pdf-to-printer';

let filePath = '/Users/xionglei/Library/Application Support/Electron/web_cache/out.pdf';

const options = {
  printer: 'PDFwriter'
};

printer
  .print(filePath, options)
  .then(res => {
    console.log(res);
  })
  .catch(err => {
    console.log(err);
  });
```
logs:
```bash
 Error: Command failed: lp /Users/xionglei/Library/Application\ Support/Electron/web_cache/out.pdf -d PDFwriter
  lp：error - “/Users/xionglei/Library/Application\ Support/Electron/web_cache/out.pdf”- No such file or directory
  
      at ChildProcess.exithandler (child_process.js:308:12)
      at ChildProcess.emit (events.js:203:13)
      at maybeClose (internal/child_process.js:1021:16)
      at Socket.<anonymous> (internal/child_process.js:430:11)
      at Socket.emit (events.js:203:13)
      at Pipe.<anonymous> (net.js:588:12) {
    killed: false,
    code: 1,
    signal: null,
    cmd: 'lp /Users/xionglei/Library/Application\\ Support/Electron/web_cache/out.pdf -d PDFwriter'
  }
```
/Users/xionglei/Library/Application Support/Electron/web_cache/out.pdf is exist

**Comment by artiebits** [comment]
Hi @xl736592857 thanks for reporting the issue. I've managed to reproduce the issue and will fix it asap

**Comment by artiebits** [comment]
@xl736592857 could you try version `pdf-to-printer@1.0.8-beta.0`, please? It contains the fix https://github.com/artiebits/pdf-to-printer/pull/68

**Comment by xl736592857** [comment]
Thank you for your reply.pdf-to-printer@1.0.8-beta.0 has fix it.

**Comment by xl736592857** [comment]
In the version 1.1.0,this error happened

**Comment by artiebits** [comment]
@xl736592857 I am sorry for this, I will check it out today

**Comment by artiebits** [comment]
Hi @xl736592857, I've just tested printing in version 1.1.0 like this `.print('/Users/arthur/Library/Application Support/Electron/web_cache/dummy.pdf')` and it printed the document.

**Comment by artiebits** [comment]
Hi @xl736592857, I've just published `pdf-to-printer@1.1.1-beta.0` which should fix it. Thanks a lot for reporting the issue!

**Comment by xl736592857** [comment]
Thank you for your reply. The problem has been solved.
